patterns/atoms/global/00-colors.mustache

```diff
@@ -0,0 +1,38 @@
+<ul class="sg-colors">
+	<li>
+		<span class="sg-swatch" style="background: #ff0000;"></span>
+		<span class="sg-label">#ff0000</span>
```

Comment:
@scottnath  - not sure how strict you want to be on indentation..? Generally i've been following 2 spaces standard...